hsingyuc/hangtime
# Create logo component

**Features**
1. Add logo for login and register page.
1. Add Google font :  Montserrat and Roboto.
1. Add padding to MuiContainer.

**Screenshot**
<img width="567" alt="螢幕快照 2020-05-11 下午11 46 42" src="https://user-images.githubusercontent.com/56378160/81610633-82870480-93e2-11ea-8604-38266d949c20.png">

<img width="529" alt="螢幕快照 2020-05-11 下午11 52 15" src="https://user-images.githubusercontent.com/56378160/81610657-8adf3f80-93e2-11ea-8eaf-26f15652b31a.png">

**Testing**
1. Check if both login and register page have logo H and small triangle.
1. Click sign in button to see if both email and password error messages show up.
